.github/workflows/build.yml

```diff
@@ -14,30 +14,29 @@ on:
 
 jobs:
     tests:
-        runs-on: ubuntu-22.04
+        runs-on: ubuntu-latest
 
         name: "Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}"
 
         strategy:
             fail-fast: false
             matrix:
-                php: [ "8.0" ]
+                php: [ "8.0", "8.1" ]
                 symfony: [ "^5.4", "^6.0" ]
-                sylius: [ "~1.11.0", "~1.12.0" ]
+                sylius: [ "~1.12.0", "~1.13.0" ]
                 node: [ "^14.17.x" ]
```

Comment:
please use node LTS version

Reply:
There is link to github actions for that branch: https://github.com/BitBagCommerce/SyliusAdyenPlugin/actions/runs/10009378477

As you can see every build using node 16 is failur. I tryied to resolve that problem but I couldn't. Maybe you have some ideas how to solve that.

Reply:
Node 20 i currently LTS version

Build are failing probably because of the outdated version of `tests/Application/package.json`

Reply:
I will upgrade `package.json` for node20 support